django_project/frontend/src/components/Table/ServerTable.tsx

```diff
@@ -71,25 +71,44 @@ const ServerTable = forwardRef(
    }: ServerTableProps, ref
   ) => {
     const { openConfirmDialog } = useConfirmDialog();
-    const [filterModel, setFilterModel] = useState({})
     if (enable.filter) {
-      columns.forEach(column => {
-        if (column.type === 'actions') {
-          if (!column.headerName) {
-            // @ts-ignore
-            column.headerName = (
-              <DataGridFilter
-                fields={columns}
-                filterModel={filterModel}
-                setFilterModel={setFilterModel}/>
-            );
-            column.headerAlign = 'right';
+      if (defaults.filters) {
```

Comment:
Could we merge filter model and parameters @zamuzakki, instead splitting it?

Reply:
I have merged those and added E2E test